pkgs/by-name/bo/borgmatic/package.nix

```
@@ -40,7 +40,7 @@ python3Packages.buildPythonApplication rec {
   # by default only 70.02% coverage is reached
   postPatch = ''
     substituteInPlace pyproject.toml \
-      --replace '--cov-fail-under=100' '--cov-fail-under=70'
+      --replace '--cov-fail-under=100' '--cov-fail-under=68'
```

```suggestion
      --replace '--cov-fail-under=100' ""
```
this is stupid, we don't care about how much coverage we reach

---

You're totally right.  I just (half-way) adopted that to the status quo, but of course this should go.